src/test/java/io/confluent/connect/jdbc/sink/integration/PostgresViewIT.java

```diff
@@ -49,35 +50,38 @@
  * Integration tests for writing to Postgres views.
  */
 @Category(IntegrationTest.class)
-public class PostgresViewIT {
+public class PostgresViewIT extends BaseConnectorIT  {
 
   private static Logger log = LoggerFactory.getLogger(PostgresViewIT.class);
 
   @Rule
   public SingleInstancePostgresRule pg = EmbeddedPostgresRules.singleInstance();
 
-  private Map<String, String> props;
   private String tableName;
   private String topic;
-  private JdbcSinkTask task;
 
   @Before
   public void before() {
+    startConnect();
+    setUpForSinkIt();
+
     tableName = "test";
     topic = tableName + "_view";
-    props = new HashMap<>();
     String jdbcURL = String
         .format("jdbc:postgresql://localhost:%s/postgres", pg.getEmbeddedPostgres().getPort());
     props.put(JdbcSinkConfig.CONNECTION_URL, jdbcURL);
     props.put(JdbcSinkConfig.CONNECTION_USER, "postgres");
     props.put(JdbcSinkConfig.TABLE_TYPES_CONFIG, "VIEW");
     props.put("pk.mode", "none");
     props.put("topics", topic);
+
+    // create topic in Kafka
+    connect.kafka().createTopic(topic, 1);
   }
 
   @After
   public void after() throws SQLException {
-    stopTask();
+   stopConnect();
```

Comment:
Same question here w/r/t `stopConnect` failures causing subsequent table cleanup logic to be skipped.

Reply:
(Same comment here: `try`/`finally` is a better way to handle this than reordering the statements)

Reply:
Right, my other comment gives my thoughts, let me know what you think!